### PR TITLE
Style and animate reviews histogram

### PR DESCRIPTION
**Pull request by avincenthill**

### CODE REVIEW

File: client/src/components/Header.jsx [hunk 1]
```diff
@@ -1,11 +1,54 @@
...
+  }
+
+  renderRatingHistogram(reviews) {
+
+    /*
+    example data - reviews should be an array of objects like these
+    {"_id":"5b918722d4a71c1897abb02f","productId":37,"reviewId":368,"username":"Parker_Nader66","stars":1,"title":"est dolore ducimus","text":"Vel molestias consequatur voluptatum vero labore dicta repellendus aut corporis. Et cupiditate minus nam. Perspiciatis dolorem omnis provident.","timestamp":"2018-02-01T17:26:28.993Z","numHelpful":965,"verifiedPurchase":true,"imageUrl":"http://lorempixel.com/640/480","__v":0}
+    */
+
+    const ratings = reviews.map((reviewObject) => {
```
Comment (soniaroh) [inline client/src/components/Header.jsx:17]: add the index parameter

**Review by soniaroh — APPROVED** [review]

File: client/src/components/Header.jsx [hunk 2]
```diff
@@ -1,11 +1,54 @@
...
+      const percentageOfRatings = (countedRatings[currentStar] / reviews.length) * 100 || 0;
+      const style = {
+        // TBD fix hardcoded arbitrary scalar 44
+        width: `${(percentageOfRatings / 100) * 44}vmin`,
+      }
+      return (
+        <React.Fragment>
+          <div className={styles.histBarContainer}>
+            <a className={styles.blueFlex}>{currentStar} star</a>
+            <a style={style} className={styles.histBar}></a>
```
Comment (mattminwoolee) [inline client/src/components/Header.jsx:42]: what's the style variable here?

Comment (mattminwoolee) [reply client/src/components/Header.jsx:42]: Oh I see it now. missed that style object above.

File: client/src/components/Review.jsx [hunk 3]
```diff
@@ -18,15 +24,15 @@ class Review extends React.Component {
...
           <div><a className={styles.gold}><b>{starArr}</b> </a><b>{review.title}</b></div>
           <div className={styles.grey}>{moment(review.timestamp).format('MMMM DD, YYYY')}</div>
-          <div className={styles.orange}><b>{isVerified}</b></div>
+          <a className={styles.lightGreyPad}> | </a><a className={styles.orange}><b>{isVerified}</b></a>
           <p className={styles.text}>{review.text}</p>
           <p className={styles.grey}>{review.numHelpful} people found this helpful</p>
-          <div className={styles.actionContainer}><button className={styles.buttonFlex}>Helpful</button> <a className={styles.lightGreyFlex}>|</a><a className={styles.greyFlex}>Comment</a><a className={styles.lightGreyFlex}>|</a><a className={styles.greyFlex}>Report Abuse</a></div>
-          <br></br>
-          <br></br>
+          <div className={styles.actionContainer}><button className={styles.buttonFlex}>Helpful</button><a className={styles.lightGreyFlex}>|</a><a className={styles.greyFlex}>Comment</a><a className={styles.lightGreyFlex}>|</a><a className={styles.greyFlex}>Report Abuse</a></div>
```
Comment (mattminwoolee) [inline client/src/components/Review.jsx:33]: This is a long branch. Recommend breaking it into a tree?

**Review by mattminwoolee — APPROVED** [review]
Looks amazing!

File: client/src/components/Review.jsx (hunk 3 above)
Comment (avincenthill) [reply client/src/components/Review.jsx:33]: Will do, thanks

**Review by michchen58 — APPROVED** [review]
echoing matt, would break down long strings into separate lines for better readability. would also be great to have more comments to explain what things do.

**Pull request merged**